nixos/modules/services/monitoring/grafana.nix

```diff
@@ -60,6 +48,26 @@ let
   # Get a submodule without any embedded metadata:
   _filter = x: filterAttrs (k: v: k != "_module") x;
 
+  # FIXME(@Ma27) remove before 23.05. This is just a helper-type
+  # because `mkRenamedOptionModule` doesn't work if `foo.bar` is renamed
+  # to `foo.bar.baz`.
+  submodule' = module: types.coercedTo
```

Comment:
Doesn't that mean that we can remove the warning for `datasources` and `dashboards`?
(I actually noticed that the warning for dashboards triggers on datasources instead. Sigh, could've sworn I've fixed it)